app/views/admin/layouts/edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= javascript_include_tag('https://unpkg.com/interactjs@1.3.3/dist/interact.min.js') %>
```

Comment:
For now this totally works, but what if that URL breaks in the near future? (Warning: I realize I'm probably being too cautious) 

Is this a valid use case to put this file under `/vendor`?

Reply:
Will read about this tomorrow!